src/test/ui/did_you_mean/issue-93210-ignore-doc-hidden.rs

```diff
@@ -0,0 +1,24 @@
+#[derive(Default)]
+pub struct A {
+    #[doc(hidden)]
+    pub hello: i32,
+    pub bye: i32,
+}
+
+#[derive(Default)]
+pub struct B {
+    pub hello: i32,
+    pub bye: i32,
+}
+
+fn main() {
+    A::default().hey;
```

Comment:
```suggestion
fn main() {
    A::default().hallo;
    A::default().hey;
```

`.hallo` may trigger similarly-named field suggestions, which may also be interesting behavior to test